.gitattributes

```diff
@@ -1 +1 @@
-* -crlf
+* eolf=lf
```

Comment:
gah, thanks. I will have to retest now that I've fixed my typos.

also, fwiw, I'm changing this branch around a lot right now just trying to find a combination that maybe does the right thing...